packages/repo-metadata-lint/cron.yaml

```diff
@@ -0,0 +1,22 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+cron:
+  - name: repo_metadata_lint
+    schedule: 0 1 * * *
+    description: repo_metadata_lint for googleapis
+    params:
+      installation:
+        id: TODO
```

Comment:
Just a reminder to come back around and catch this :)

Reply:
```suggestion
        id: 21068138
```